app/Nova/Metrics/PrimaryTeam.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace App\Nova\Metrics;
+
+use App\Team;
+use App\User;
+use App\Attendance;
+use Illuminate\Http\Request;
+use Laravel\Nova\Metrics\Value;
+use Illuminate\Support\Facades\DB;
+
+class PrimaryTeam extends Value
+{
+    /**
+     * Calculate the value of the metric.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return mixed
+     */
+    public function calculate(Request $request)
+    {
+        $gtid = User::where('id', $request->resourceId)->first()->gtid;
+        \Log::info('gt'.$gtid);
+        $teams = Attendance::where('gtid', $gtid)
+            ->where('attendable_type', 'App\Team')
+            ->groupBy('attendable_id')
+            ->select('attendable_id', DB::raw('count(*) as count'))
+            ->get()->toArray();
+
+        $max = 0;
+        $maxTeamIDs = [];
+        foreach ($teams as $item) {
+            if ($item['count'] == $max) {
+                $maxTeamIDs[] = $item['attendable_id'];
+            } elseif ($item['count'] > $max) {
+                $max = $item['count'];
+                $maxTeamIDs = [$item['attendable_id']];
+            }
+        }
+
+        if (count($maxTeamIDs) == 0) {
+            return $this->result('No teams or attendance');
+        } else {
+            $names = Team::whereIn('id', $maxTeamIDs)->get()->pluck('name')->toArray();
+
+            return $this->result(implode(', ', $names));
+        }
+    }
+
+    /**
+     * Get the ranges available for the metric.
+     *
+     * @return array
+     */
+    public function ranges()
+    {
+        return [];
```

Comment:
I kind of want to filter this by date range and default to ~2 months as affiliations can change over time.

Reply:
Added in a930d04